src/components/ens-registration/RegistrationAvatar/RegistrationAvatar.tsx

```diff
@@ -74,8 +73,8 @@ const RegistrationAvatar = ({
     }
   }, [initialAvatarUrl, avatarUpdateAllowed]); // eslint-disable-line react-hooks/exhaustive-deps
 
-  // We want to allow avatar state update when the screen is first focussed.
-  useFocusEffect(useCallback(() => setAvatarUpdateAllowed(true), []));
+  // We want to allow avatar state update when the screen is first focused.
+  useEffect(() => setAvatarUpdateAllowed(true), [setAvatarUpdateAllowed]);
```

Comment:
don't we want to set it back to true when the screen comes back in focus tho?

Reply:
why do we even need the condition `avatarUpdateAllowed`? maybe i'm missing something but i don't think we need to re-set the avatar at any time (i.e. in the useEffect) except for when it is explicitly done in `onChangeImage`

Reply:
also is there a reason why we don't use the image_url property that i just used in my most recent commit? it seems to be higher res

Reply:
This screen is not lazy, so it will render when the intro screen renders. `useFocusEffect` mitigates this. If we don't have a `useFocusEffect`, then the user will not be able to upload another avatar if they modify the registration name.

Reply:
> also is there a reason why we don't use the image_url property

yeah, it's because we don't need high res images for the avatar component (for NFT avatars), since we use a eip155 string for the `avatar`/`cover` text record

Reply:
i think the reason why the useFocusEffect might not be working properly here is because the avatar/cover component gets focused/unfocused multiple times when the user interacts with it (ie uploads something).

Reply:
i think i can get the intended effect from the `useEffect`s i added by including the registration name in the dependency array